vmm/src/device_manager.rs

```diff
@@ -564,6 +584,9 @@ pub struct DeviceManager {
     // Backends that have been spawned
     vhost_user_backends: Vec<ActivatedBackend>,
 
+    // Counter to keep track of the consumed device IDs.
+    device_id_cnt: Wrapping<usize>,
+
```

Comment:
Nit: Any reason why this is moved up?

Reply:
Oh that's because it's not PCI specific anymore, so I thought moving it up would make this clearer. But I can move it back if you want.